compositor/view.c

```diff
@@ -93,9 +93,13 @@ void wc_view_damage(struct wc_view *view, pixman_region32_t *damage) {
 
 	for (size_t i = 0; i < 4; i++) {
 		struct wlr_output *output = outputs[i];
+		struct wlr_box *output_box =
+				wlr_output_layout_get_box(view->server->output_layout, output);
+
 		if (output) {
-			struct wlr_box view_output_geo = {.x = view->geo.x - output->lx,
-					.y = view->geo.y - output->ly,
+			// TODO replace lx and ly with xdg-output
```

Comment:
Ok tested it out, and not only does it work but these values aren't necessary here at all. Can you just remove the box access altogether then?